courses/fundamentals_of_ada/090_overloading.rst

```diff
@@ -320,59 +326,6 @@ Which statement is not legal?
    C. Use of :ada:`Top` resolves ambiguity
    D. When overloading subprogram names, best to not just switch the order of parameters
 
-===================
-Visibility Issues
-===================
-
-----------
-Examples
-----------
-
-.. include:: examples/090_overloading/visibility_issues.rst
-
-:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/090_overloading.html#visibility-issues`
-
------------------------------------
-Inherently Ambiguous Declarations
```

Comment:
This is important - details how overloading relies on base type not subtypes.
Maybe add it back and add more explanation?

Reply:
Agreed!